chord_metadata_service/restapi/urls.py

```diff
@@ -91,7 +91,4 @@
     path('public_overview', public_overview, name='public-overview',),
     path('public_dataset', public_dataset, name='public-dataset'),
     path('public_rules', public_rules, name='public-rules'),
-
-    # uncensored endpoint for beacon search using fields from config.json
-    path('beacon_search', individual_views.BeaconListIndividuals.as_view(), name='beacon-search'),
```

Comment:
is this because Beacon will now use the regular search endpoint?

Reply:
that's the idea - I'll have to implement a system for it to get its own token though.